rpm/SPECS/sth.spec

```
@@ -160,6 +160,11 @@ rm -rf $RPM_BUILD_ROOT
 %{_install_dir}
 
 %changelog
+* Fri Oct 28 2022 Alvaro Vega <alvaro.vegagarcia@telefonica.com> 2.10.0
```

We are going to freeze release next Wednesday, so:

```suggestion
* Web Nov 02 2022 Alvaro Vega <alvaro.vegagarcia@telefonica.com> 2.10.0
```